src/uproot/source/fsspec.py

```diff
@@ -155,3 +163,16 @@ def closed(self) -> bool:
         otherwise.
         """
         return False
+
+
+class FSSpecLoopExecutor(uproot.source.futures.Executor):
+    def __init__(self, loop: asyncio.AbstractEventLoop):
+        self.loop = loop
+
+    def submit(self, coroutine, *args) -> asyncio.Future:
```

Comment:
This is actually returning a `concurrent.futures.Future` which is subtly different than an asyncio Future. I guess mypy would flag this if we were actively testing with it.